source/includes/authenticated_api/_signatures.md.erb

```diff
@@ -132,6 +132,22 @@ The JSON response has a single "signature" object, which may include the followi
 <div></div>
 
 ### Create
+> POST /api/v1/petitions/no-taxes-on-tea/signatures/
+
+```json
+{
+  "signature": {
+    "first_name": "Foo",
+    "last_name": "Bar",
+    "email": "foo@bar.com",
+    "postcode": "10010",
+    "phone_number": "123-456",
```

Comment:
nitpick: this isn't a valid US phone number
(We could use `555-555-5555` instead)